src/shared/components/ncTable/partials/rowTypePartials/TextLineForm.vue

```diff
@@ -28,6 +28,11 @@ export default {
 	computed: {
 		localValue: {
 			get() {
+				if (this.value === null) {
+					const newValue = this.column?.textDefault ? this.column.textDefault : ''
```

Comment:
I think you could use the [`??` operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) - aka the nullish coalescing operator as it's officially called.